storage/quota/prjquota.go

```diff
@@ -321,19 +321,30 @@ func (quota *PrjQuotaDriver) setDevLimit(dir string, devID uint64) (uint64, erro
 		return limit, nil
 	}
 
```

Comment:
Do we need to update the function in grpquota, it seems like a common interface function implement.